Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -1194,18 +1191,6 @@ void Renderer::Swap(u32 xfbAddr, u32 fbWidth, u32 fbStride, u32 fbHeight, const
 
       auto xfb_rect = texture_config.GetRect();
 
-      // It's possible that the returned XFB texture is native resolution
-      // even when we're rendering at higher than native resolution
-      // if the XFB was was loaded entirely from console memory.
-      // If so, adjust the rectangle by native resolution instead of scaled resolution.
-      const u32 native_stride_width_difference = fbStride - fbWidth;
-      if (texture_config.width == xfb_entry->native_width)
-        xfb_rect.right -= native_stride_width_difference;
-      else
-        xfb_rect.right -= EFBToScaledX(native_stride_width_difference);
-
-      m_last_xfb_region = xfb_rect;
```

Comment:
I think you might have deleted a little too much.  We probably want to keep this line

Reply:
Hmm. The "rectangle" is essentially determined by the xfb address and width, so the scaling should happen in TextureCacheBase::StitchXFBCopies() (as well as sub-rectangles if only part of an XFB copy is scanned out). Do you know of any cases where this is broken?

Also, assuming the stride is related to the width is incorrect, the stride should be double the width in interlaced modes.

Reply:
I can go test it but my assumption is `DumpCurrentFrame` is broken since it uses the `m_last_xfb_region` which is not being set any more.

Reply:
Ohh sorry, I misread. Well, we can just drop it entirely, as the texture will always contain the full XFB.

Reply:
No problem.  That's great to hear!